src/api/test-controller.js

```diff
@@ -107,10 +109,9 @@ export default class TestController {
             throw err;
         }
 
-        this.executionChain       = this.executionChain.then(() => this.testRun.executeCommand(command, callsite));
         this.callsiteWithoutAwait = callsite;
 
-        return this._createExtendedPromise(this.executionChain, callsite);
+        return this._createExtendedPromise(this.testRun.executeCommand(command, callsite), callsite);
```

Comment:
I'm not sure how error propagation will work without execution chain (if it will work at all):
```
await t
         .click('1')
         .click('2')     // error occurs here, will it reject click(3)? Will it propagate? Because now they are not connected with promise chain
         .click('3');
```